web/src/components/Cart/Cart.js

```diff
@@ -1,5 +1,5 @@
 import * as Dialog from '@radix-ui/react-dialog'
-import { ShoppingCart } from 'react-feather'
+import { ShoppingCart, Trash } from 'react-feather'
```

Comment:
Could we use [Trash2](https://feathericons.com/?query=trash-2)? I kinda like the style of that one a little more
```suggestion
import { ShoppingCart, Trash2 } from 'react-feather'
```

Reply:
Sounds good to me.